software/controller/lib/debug/var_cmd.cpp

```diff
@@ -30,13 +30,13 @@ ErrorCode VarHandler::Process(Context *context) {
   switch (context->request[0]) {
```

Comment:
I think it would be cleaner to do something like
```
auto subcmd = static_cast<Subcommand>(context->request[0]);
switch (subcmd) {
```
and avoid the casts in the switch statements.